src/containers/Balance/CurrentBalanceContainer.tsx

```diff
@@ -1,16 +1,25 @@
 import React, {ReactElement} from "react";
 import "./balance.scss";
+import {INodeBalance} from "../Dashboard/NodeInterface";
 
-export const CurrentBalanceContainer = (): ReactElement => {
+export interface ICurrentBalanceContainerProps {
+    balance: INodeBalance | null;
+}
+
+export const CurrentBalanceContainer: React.FC<ICurrentBalanceContainerProps> = ({
+    balance,
+}: ICurrentBalanceContainerProps): ReactElement => {
     return (
         <div className="container flex-column">
             <div className="upper">
                 <label>Current balance</label>
             </div>
 
             <div className="lower balance">
-                <h2>122.4589 FIL</h2>
-                <p className="yellow">+ 0.0189 FIL (0,0318%)</p>
+                <h2>{balance && balance.currentBalance} FIL</h2>
```

Comment:
It would be better if you have condition above for this i.e.
```
if (!balance) {
  return "Loading";
}
```

later we will easier replace this with real loading indicator and have better UX (currently you have nothing and then something pops up which will be more noticable when not using localhost)